components/BottomNavigation/src/private/MDCBottomNavigationItemView.h

```diff
@@ -35,6 +35,7 @@
 @property(nonatomic, strong) UIImage *selectedImage;
 
 @property(nonatomic, strong) UIColor *badgeColor UI_APPEARANCE_SELECTOR;
+@property(nonatomic, strong) UIColor *badgeTextColor;
```

Comment:
Since this is a pass-through to UILabel.textColor, please make this as `null_resettable`.

Reply:
Done